sql/core/src/main/scala/org/apache/spark/sql/scripting/SqlScriptingExecutionContext.scala

```diff
@@ -89,4 +96,6 @@ class SqlScriptingExecutionFrame(
  * @param label
  *   Label of the scope.
  */
-class SqlScriptingExecutionScope(val label: String)
+class SqlScriptingExecutionScope(val label: String) {
+  val variables = new mutable.HashMap[String, VariableDefinition]
```

Comment:
shall we simply put the `TempVariableManager` here? instead of putting a map and linking it to `SqlScriptingVariableManager`